modules/core/04-channel/keeper/grpc_query_test.go

```diff
@@ -663,6 +663,22 @@ func (suite *KeeperTestSuite) TestQueryPacketCommitment() {
 			},
 			false,
 		},
+		{
+			"commitment not found",
+			func() {
+				path := ibctesting.NewPath(suite.chainA, suite.chainB)
+				path.Setup()
+				expCommitment = []byte("hash")
+				suite.chainA.App.GetIBCKeeper().ChannelKeeper.SetPacketCommitment(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID, 1, expCommitment)
+				
+				req = &types.QueryPacketCommitmentRequest{
+					PortId:    path.EndpointA.ChannelConfig.PortID,
+					ChannelId: path.EndpointA.ChannelID,
+					Sequence:  2,
+				}
+			},
+			false,
+		},
```

Comment:
I added this test because the one above, called "channel not found" was not actually testing it in my understanding (since no check for `HasChannel` was performed), but it was actually hitting the line that checks for the commitment. So I created this to cover the scenario, by creating the channel and changing the Sequence parameter on the request, so that it passes the `HasChannel` check but fails after that.